packages/writing-style/styles/commercetools/Acronyms.yml

```diff
@@ -31,6 +31,7 @@ exceptions:
   - HTML
   - HTTP
   - HTTPS
+  - ICU
```

Comment:
Where is that used and what is it. I know only Intensive Care Unit as ICU ;-)

Reply:
This refers to the "internationalization" syntax (`Intl`). It's what we use in the MC.

https://unicode-org.github.io/icu/userguide/icu/

Reply:
Not sure if ICU is that well known to not explain it the first time it occurs.  But I would leave it as "allowed" because explaining it as "International Components for Unicode" does not make anything clearer, too :-)